### ISSUE TITLE
Why the validation loss is often smaller than the training loss?

### ISSUE THREAD

**Issue by benjamin3344** [pytorch/fairseq#3479]
As is shown in many cases, e.g...  why the validation loss is often way smaller than the training loss?

https://github.com/pytorch/fairseq/issues/3137

**Comment by RuABraun** [comment]
Because there are a lot of different regularizations that make training hard which are off in eval mode. This is not unexpected.

**Comment by benjamin3344** [comment]
> Because there are a lot of different regularizations that make training hard which are off in eval mode. This is not unexpected.

Thank you. I think the same. I try to find out in which part the regularization is introduced, but I can't. There seems no dropout or batch norm in the codes I used.
